File: DataStructure-and-Algorithm/Search/Linear_Search/linear_search.cpp
#include<iostream>
using namespace std;

int main() 
{
	cout<<"Enter The Size Of Array: ";
	int size;
	cin>>size;

	int array[size], key,i,loc = -1;
	
	for(int j=0;j<size;j++)
	{
		cout<<"Enter "<<j+1<<" Element: ";
		cin>>array[j];
	}
	
	cout<<"array[ ";
	for(int a=0;a<size;a++)
	{
		cout<<array[a]<<", ";
	}
	cout<<"]\n";
	
	cout<<"Enter Value To Search in Array : ";
	cin>>key;
	for(i=0;i<size;i++)
	{
		if(key==array[i])
		{
			loc = i;
			break;
		}
	}
	if(loc!=-1)
		cout<<"Value Found At Index Number : "<<loc<<endl;
	else
		cout<<"Value not found in Array"<<endl;
   return 0;
}

File: DataStructure-and-Algorithm/Search/Linear_Search/linear_search.java
import java.util.*;
public class linear_search{
    public static void main(String[] args){
        Scanner sc = new Scanner(System.in);
        System.out.print("Enter the size of array : ");
        int size = sc.nextInt();

        int array[] = new int[size];

        for(int j=0;j<size;j++)
        {
            System.out.print("Enter "+(j+1)+" Element: ");
            array[j]=sc.nextInt();
        }

        System.out.print("array[ ");
        for(int i = 0;i<size;i++){
            System.out.print(array[i]+", ");
        }
        System.out.println("]");

        int loc = -1;
        System.out.print("Enter Value To Search in Array : ");
        int key = sc.nextInt();
        for(int i=0;i<size;i++)
        {
            if(key==array[i])
            {
                loc = i;
                break;
            }
        }
        if(loc!=-1)
            System.out.println("Value Found At Index Number : "+loc);
        else
            System.out.println("Value not found in Array");
    }
}